### ISSUE TITLE
Pupil player can't load recording from Pupil Mobile

### ISSUE THREAD

**Issue by teresaCB** [pupil-labs/pupil-mobile-app#29]
Pupil Player fails to load the recording data from Pupil Mobile. I drop the folder (that I previously extracted from the android) into Pupil Player but it never loads it. Is there a previous formatting that needs to be done to be able to load it?

See picture of the folder files.
<img width="495" alt="screenshot 2018-11-03 21 37 44" src="https://user-images.githubusercontent.com/34170792/47960113-4df6e900-dfb2-11e8-991e-1426db1d7e29.png">

**Comment by teresaCB** [comment]
Yes, it does have a world camera and I can successfully see through it in the pupil mobile app. Any ideas why the file isn't there?

**Comment by willpatera** [comment]
What version of Pupil Mobile are you using?

**Comment by teresaCB** [comment]
0.31.0-revert

**Comment by teresaCB** [comment]
I have updated Pupil mobile app to its beta version and now I can see the world camera videos in the file. I tried to now load the data in Pupil Player, and still won't load it (see the log below).
![screenshot 2018-11-05 22 45 59](https://user-images.githubusercontent.com/34170792/48047332-977b3b80-e14c-11e8-9bdc-a315401663d5.png)

2018-11-05 22:34:31,747 - MainProcess - [INFO] os_utils: Disabled idle sleep.
2018-11-05 22:34:33,238 - player - [ERROR] player_methods: No valid dir supplied (/Applications/Pupil Player.app/Contents/MacOS/pupil_player)
2018-11-05 22:34:42,303 - player - [INFO] launchables.player: Starting new session with '/Users/teresa/recordings/20181105222359132'
2018-11-05 22:34:42,312 - player - [INFO] player_methods: Updating meta info
2018-11-05 22:34:42,313 - player - [INFO] player_methods: Converting Pupil Mobile recording to v0.9.4 format
2018-11-05 22:34:42,314 - player - [INFO] player_methods: Creating "World Camera_timestamps.npy"
2018-11-05 22:34:42,322 - player - [INFO] player_methods: Renaming "World Camera.mp4" to "World Camera.mp4"
2018-11-05 22:34:42,323 - player - [INFO] player_methods: Creating "Audio_timestamps.npy"
2018-11-05 22:34:42,324 - player - [INFO] player_methods: Renaming "Audio.mp4" to "Audio.mp4"
2018-11-05 22:34:42,324 - player - [INFO] player_methods: Creating "Left eye_timestamps.npy"
2018-11-05 22:34:42,325 - player - [INFO] player_methods: Renaming "Left eye.mjpeg" to "Left eye.mjpeg"
2018-11-05 22:34:42,326 - player - [INFO] player_methods: Creating "Right eye_timestamps.npy"
2018-11-05 22:34:42,326 - player - [INFO] player_methods: Renaming "Right eye.mjpeg" to "Right eye.mjpeg"
2018-11-05 22:34:42,327 - player - [INFO] player_methods: Creating "pupil_data"
2018-11-05 22:34:42,327 - player - [INFO] player_methods: Updating meta info
2018-11-05 22:34:42,328 - player - [INFO] player_methods: Updating recording from v0.9.4 to v0.9.13
2018-11-05 22:34:42,329 - player - [INFO] player_methods: Updating meta info
2018-11-05 22:34:42,329 - player - [INFO] player_methods: Updating recording from v0.9.13 to v0.9.15
2018-11-05 22:34:42,330 - player - [INFO] player_methods: Updating meta info
2018-11-05 22:34:42,330 - player - [INFO] player_methods: Updating recording from v0.9.15 to v1.3
2018-11-05 22:34:42,331 - player - [INFO] player_methods: Updating meta info
2018-11-05 22:34:42,331 - player - [INFO] player_methods: Updating recording from v1.3 to v1.4
2018-11-05 22:34:42,332 - player - [INFO] player_methods: Updating meta info
2018-11-05 22:34:42,332 - player - [INFO] player_methods: Checking for world-less recording
2018-11-05 22:34:42,333 - player - [ERROR] launchables.player: Could not generate world timestamps from eye timestamps. This is an invalid recording.
2018-11-05 22:45:06,769 - MainProcess - [INFO] os_utils: Re-enabled idle sleep.

**Comment by willpatera** [comment]
@teresaCB could you also let me know what mobile device you are using?

**Comment by teresaCB** [comment]
Moto Z2 play with the 7.1.1 android version

**Comment by papr** [comment]
The problem is that the video+time files are not recognized as eye/world cameras. Is it possible that you using a DIY-headset? Please rename them as following:
```
Right eye.*    -> Pupil Cam1 ID0.*
Left eye.*     -> Pupil Cam1 ID1.*
World Camera.* -> Pupil Cam1 ID2.*
```

**Comment by teresaCB** [comment]
That fixed the problem!!

I changed the names on the app just to make them more intuitive, but I guess they'll have to stay like that.

Thank you so much
